Content/ContentTypeResolver/LinkResolver.php

```diff
@@ -76,10 +76,9 @@ private function getViewData(PropertyInterface $property): array
         return $result;
     }
 
-    private function getContentData(PropertyInterface $property): ?string
+    private function getContentData(PropertyInterface $property, string $locale): ?string
     {
         $value = $property->getValue();
-        $locale = $property->getStructure()->getLanguageCode();
```

Comment:
This change looks strange for me. If I understand you correctly currently not the correct `Locale` is set on the Structure LanguageCode? Just the LanguageCode? Think that would make not only for Links Content Type a problem 🤔